code/game/gamemodes/changeling/changeling_items.dm

```diff
@@ -47,6 +47,9 @@
 			host.drop_from_inventory(src)
 		QDEL_IN(src, 1)
 
+/obj/item/weapon/melee/arm_blade/iscrowbar()
+	return TRUE
+

```

Comment:
```suggestion

/obj/item/weapon/melee/arm_blade/resolve_attackby(atom/A, mob/living/user, var/click_parameters)
	if(istype(A,/turf/simulated/floor) && user.a_intent != I_HELP)
		return
	else
		..()
```

Reply:
Makes crowbarring tiles with the armblade only work on help, to make fights look less silly.